### ISSUE TITLE
[SideNavigationGroup]: A11y state as ‘expanded/collapsed’ is not read out to the user

### ISSUE THREAD

**Issue by AlekseiSch** [SAP/ui5-webcomponents#11421]
### Bug Description

With Voiceover enabled, on accessing ‘Group 1’ or 'Group 2' item, the state as ‘expanded/collapsed’ is not read out to the user.
Reproduced on MacOs with VoiceOver, also reproducible on iOS, iPhone.

### Affected Component

SideNavigationGroup

### Expected Behaviour

With Voiceover enabled, state should be read out to screen reader users

### Isolated Example

https://sap.github.io/ui5-webcomponents-react/v2/?path=/story/layouts-floorplans-sidenavigation--grouped

### Priority

None

### UI5 Web Components Version

2.9.0

### Browser

Safari

### Operating System

MacOs, iOS

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by AlekseiSch** [comment]
Hi, any updates on that?
